script/app.js

```diff
@@ -0,0 +1,67 @@
+let bookCollection = [];
+let newBook = null;
+
+const updateLocalStorage = () => {
+  localStorage.setItem('bookCollection', JSON.stringify(bookCollection));
+};
+
+const removeBook = (bookId, storeBooks) => {
+  const book = document.getElementById(`${bookId}`);
+  storeBooks.removeChild(book);
+  bookCollection = bookCollection.filter((Object) => Object.id !== bookId);
+  updateLocalStorage();
+};
+
+const addBook = (title, author) => {
+  const newBook = {
+    id: String(bookCollection.length),
+    title,
+    author,
+  };
+  bookCollection.push(newBook);
+  const storeBooks = document.querySelector('#storeBooks');
```

Comment:
[OPTIONAL] Kindly clear the input fields after pressing add button for a better user experience.

Reply:
@gemmen29 Thank you for the thorough review provided. We will make the optional changes right away.

Reply:
@gemmen29 Optional changes have been implemented https://github.com/Tadesse-Alemayehu/Awesome-books/pull/1/commits/ae078dd12128336a98b2b96f3d729669306fe509. If there is anything we can improve on, please let us know.

Reply:
Good job.

Reply:
Thank you ✌.